baseapp/baseapp_test.go

```diff
@@ -33,11 +26,18 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
 	"github.com/cosmos/cosmos-sdk/x/auth/migrations/legacytx"
+	"github.com/gogo/protobuf/jsonpb"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	abci "github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/log"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
 )
 
 var (
-	capKey1 = sdk.NewKVStoreKey("key1")
-	capKey2 = sdk.NewKVStoreKey("key2")
+	capKey1    = sdk.NewKVStoreKey("key1")
+	capKey2    = sdk.NewKVStoreKey("key2")
+	randSource = int64(3920758213583)
```

Comment:
Where'd this number come from?

Reply:
It was already in the method `setupBaseAppWithSnapshots` so I just took it from there. Do you prefer to change it to something readable like `123456789`

Reply:
Was just curious.